apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-intent/use-site-intent.js

```diff
@@ -1,27 +1,14 @@
-import apiFetch from '@wordpress/api-fetch';
-import { useState, useEffect, useCallback } from '@wordpress/element';
-
 // FIXME: We can use `useSiteIntent` from `@automattic/data-stores` and remove this.
 // https://github.com/Automattic/wp-calypso/pull/73565#discussion_r1113839120
 const useSiteIntent = () => {
-	const [ siteIntent, setSiteIntent ] = useState( undefined );
-	const [ siteIntentFetched, setSiteIntentFetched ] = useState( false );
-
-	const fetchSiteIntent = useCallback( () => {
-		apiFetch( { path: '/wpcom/v2/site-intent' } )
```

Comment:
When did the endpoint `/wpcom/v2/site-intent` become unavailable?
I was wondering because this hook itself seems to have been in use since March 2022.

Reply:
It's unavailable on the Jetpack site at the beginning 😓

Reply:
😇

Reply:
Its available from simple sites and requests to the public api. However, in the editor atomic sites make these requests directly to the atomic sites themselves where this endpoint doesn't exist on its own.

Reply:
Yes, that's true 👍